### ISSUE TITLE
[FEATURE_REQUEST] API or Kubernetes Discovery

### ISSUE THREAD

**Issue by m4dm4rtig4n** [Lissy93/dashy#429]
### Is your feature request related to a problem? If so, please describe.

Hello,

I would like to know if there are plans to integrate APIs one day or another or to do service discovery via Kubernetes?

### Describe the solution you'd like

Just an API endpoint to write config :)

### Priority

Low (Nice-to-have)

**Comment by Lissy93** [comment]
Heya @m4dm4rtig4n 
Thanks for your feature request :) There are indeed plans to build out an API (for reading + writing to config), it's num 10  on the current [Roadmap](https://github.com/Lissy93/dashy/discussions/405). But I don't yet have a date for when it will be completed by, likely not until Spring.

I'm still quite new to Kubernetes, is the requirement you need just GET / PUT / POST API endpoints to read / add / update app ID addresses in the config? Or have I completely misunderstood the question?
